src/test/scala/org/graphframes/lib/PageRankSuite.scala

```diff
@@ -27,6 +29,11 @@ class PageRankSuite extends SparkFunSuite with GraphFrameTestSparkContext {
 
   val n = 100
 
+  private def isLaterVersion(minVer: String): Boolean = {
+    val minVers = minVer.split('.').map(_.toInt)
+    spVers.zip(minVers).forall { case (spV, minV) => spV >= minV }
```

Comment:
Is this logic correct?  It becomes easier when you just look at the major and minor version numbers.  You could also lift this: https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/util/VersionUtils.scala